README.md

```diff
@@ -1,3 +1,26 @@
+# My Event Listeners
+
+- `wheel up`
+  - remove navbar darkmode
+
+- `wheel down`
+
+  - add navbar darkmode
+
+- `dblclick`
+  - double click picture to enter fullscreen
+
+- `onkeyup`
+
+  - exit picture fullscreen
+
+- `mouseover enter`
+
+  - **add** `hover` class to navbar logo
+
+- `mouseover leave`
+  - **removes** `hover` class to navbar logo
+
```

Comment:
I like how you annotated your event listeners in the readme.  nice work with that.